src/index.js

```diff
@@ -1858,6 +1866,25 @@ const initialize = async () => {
     networkDiv.innerHTML = networkId;
   }
 
+  function handleScrollTo() {
+    if (!scrollTo) {
+      return;
+    }
+
+    scrollToHandled = true;
+
+    console.log('Attempting to scroll to element with ID:', scrollTo);
+
+    const scrollToElement = document.getElementById(scrollTo);
+
+    if (!scrollToElement) {
+      console.log('Cannot find element with ID:', scrollTo);
```

Comment:
`console.warn` maybe better in this case.

Reply:
Done.